installing/installing_openstack/installing-openstack-user.adoc

```diff
@@ -35,7 +35,6 @@ include::modules/installation-osp-verifying-external-network.adoc[leveloffset=+1
 include::modules/installation-osp-accessing-api.adoc[leveloffset=+1]
 include::modules/installation-osp-accessing-api-floating.adoc[leveloffset=+2]
 include::modules/installation-osp-describing-cloud-parameters.adoc[leveloffset=+1]
-include::modules/installation-user-infra-generate.adoc[leveloffset=+1]
```

Comment:
This file still says that it's used in this assembly. Please also update the module.

Reply:
🥇